### ISSUE TITLE
Remember expanded and collapsed folders

### ISSUE THREAD

**Issue by melvin4u445** [kirillt/ark-navigator#55]
This is more of an improvement than a bug. 
It is a few extra steps each time to open the root folders again after pressing the compass button and returning back to the Manage Folders tab.

Ideally, the app remembers the expanded and collapsed folders and displays it when the user returns back to the view
 
https://user-images.githubusercontent.com/17034768/132106258-bcfc7217-462c-4c52-ae09-6b29fa002799.mp4

**Comment by kirillt** [comment]
Agree, that always takes couple of seconds. But I am not sure it should blindly remember last state since a user can, technically, create huge hierarchy even with just 3 levels (devices/roots/favorites). Maybe we should expand levels automatically basing on how big they are.

**Comment by melvin4u445** [comment]
Would there be a performance issue on the device if we remember the states and the hierarchy is too big?

**Comment by kirillt** [comment]
Technically it isn't difficult and performance would not suffer neither.
